client/starwhale/core/dataset/cli.py

```diff
@@ -29,6 +29,8 @@ def dataset_cmd(ctx: click.Context) -> None:
 )
 @click.option("--append", is_flag=True, help="Only append new data")
 @click.option("--append-from", default=LATEST_TAG, help="Append from dataset version")
+@click.option("--runtime", default="", help="runtime uri")
+@click.option("--runtime-restore", is_flag=True, help="Force to restore runtime")
```

Comment:
Do users want to restore the runtime env when they are building a dataset?

Reply:
`runtime-restore`  scenario: 
- Users want to build the dataset from one standalone runtime URI.
- But the runtime's workdir is empty, in another word the runtime cannot be activated directly. Then create a virtual environment and install dependencies by restoring command. Activate it when run dataset build.

`runtime-restore` command can be executed repeatedly and fastly with the cache which is a shortcut argument for model/dataset/eval that is using the runtime.  If ignore `runtime-restore` argument, users should run `swcli runtime restore` command in advance.

Reply:
I get id :ok_hand: